api/src/main/java/com/epam/pipeline/manager/user/UserApiService.java

```diff
@@ -110,6 +111,10 @@ public PipelineUser updateUserRoles(Long id, List<Long> roles) {
         return new ArrayList<>(userManager.loadAllUsers());
     }
 
+    public List<UserInfo> loadUsersInfo() {
```

Comment:
We still need @PreAuthorize annotation here hasRole('ADMIN') OR hasRole('ROLE_USER')